frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js

```diff
@@ -13,15 +13,20 @@ const mapStateToProps = state => ({
 const SampleDetailsProcessMeasurements = ({processMeasurements, usersByID, protocolsByID,}) => {
     const columns = [
       {
-        title: '',
+        title: 'ID',
```

Comment:
I would change this to "Sample Process ID" to make it clearer of the ID this designate.